website/blog/2025-04-14-release-1.18.md

```diff
@@ -0,0 +1,390 @@
+---
+title: Podman Desktop 1.18 Release
+description: Podman Desktop 1.18 has been released!
+slug: podman-desktop-release-1.18
+authors: SoniaSandler
+tags: [podman-desktop, release, podman]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.18/banner.png
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+Podman Desktop 1.18 Release! 🎉
+
+<!-- ![podman-desktop-hero-1.18](/img/blog/podman-desktop-release-1.18/banner.png) -->
+
+Podman Desktop 1.18 is now available! [Click here to download it](/downloads)!
+
+This release brings exciting new features and improvements:
+
+- **Refining the Docker Compatibility feature**: The Docker Compatibility page has been moved out of experimental mode, and new supporting components such as podman-mac-helper and mac podman socket failures notifications have been added to help users set it up
+- **Switching Kubernetes namespace**: Switching between Kubernetes namespaces within Podman Desktop is now available
+- **Jobs in Kubernetes**: You can now see Kubernetes jobs in our growing Kubernetes section of Podman Desktop
+- **Experimental statusbar providers enhancments**: Statusbar providers can be pinned and un-pinned based on user preferences and the provider status icons have been updated to be easier to understand
+
+---
+
+## Release Details
+
+### Docker Compatibilty mode page is out of experimental mode 🎉
+
+The experimental Docker Compatibility page that was introduced in [v1.13](https://podman-desktop.io/blog/podman-desktop-release-1.13#docker-compatibility-mode-page) has been moved out of experimental mode, so now everything related to Docker compatibility can be found in one place for you to use. In addition, new Docker compatibility related notifications have been added to make the set up process easier and less guesswork when running into problems.
+
+<ThemedImage
+alt="Docker Compatibilty podman mac helper notification"
+sources={{
+    light: require('./img/podman-desktop-release-1.18/mac-helper-light.png').default,
+    dark: require('./img/podman-desktop-release-1.18/mac-helper-dark.png').default,
+  }}
+/>
+
+<ThemedImage
+alt="Docker Compatibilty docker socket notification"
+sources={{
+    light: require('./img/podman-desktop-release-1.18/docker-socket-light.png').default,
+    dark: require('./img/podman-desktop-release-1.18/docker-socket-dark.png').default,
+  }}
+/>
+
+### Switching Kubernetes namepaces
+
+Until now, only switching Kubernetes context was supported in Podman Dekstop. But no more, as now users are also able to switch between namespaces from within Podman Desktop without having to do anything outside the app. The selection does not modify Kube config, and is presistent across the app, so if you select a specific namespace in one place, it will be used everywhere else in Podman Desktop.
```

Comment:
```suggestion
Until now, only switching Kubernetes context was supported in Podman Dekstop. But no more, now you can switch between namespaces from within Podman Desktop without having to do anything outside the app. The selection does not modify your Kube config, and is persistent across the app, so if you select a specific namespace in one place, it will be used everywhere else in Podman Desktop.
```